### PR TITLE
swaps - explainer updates

### PR DESCRIPTION
**Pull request by skylarbarrera**
Fixes TEAM2-204, TEAM2-211, TEAM2-182

## What changed (plus any additional context for devs)
changed alert to an explainer, update copy, tweaked some rendering logic in explain sheet 

## PoW (screenshots / screen recordings)

unverified explainer: 
https://cloud.skylarbarrera.com/Screen-Recording-2022-06-29-14-43-00.mp4

updated copy + styles: https://cloud.skylarbarrera.com/Screen-Shot-2022-06-29-16-14-35.80.png
android: 
Screen Shot 2022-06-29 at 4 51 24 PM

no token explainer: (this still needs copy updates but I updated to the current state of the ticket for now

iOS: https://cloud.skylarbarrera.com/Screen-Shot-2022-06-29-16-50-44.37.png
android: 
Screen Shot 2022-06-29 at 4 49 33 PM

copy update: https://cloud.skylarbarrera.com/Screen-Shot-2022-06-29-18-05-46.11.png

Output disabled copy update:
https://cloud.skylarbarrera.com/Screen-Shot-2022-06-29-18-03-48.11.png

## Dev checklist for QA: what to test
explainer sheet should work as shown in video, other explainer sheets should not be affected (big ones here are swap settings ones since they use the tweaked logic)

## Final checklist

- [x] Assigned individual reviewers?
- [x] Added labels?
- [x] Added e2e tests? if not please specify why
- [x] If you added new files, did you update the CODEOWNERS file?

### CODE REVIEW

**Review by estebanmino — COMMENTED** [review]
android has no margin between buttons
image

**Review by estebanmino — COMMENTED** [review]
same happening with newest changes 
image

also feels like the line height between first and second paragraph is different?

image

image

File: src/screens/ExplainSheet.js [hunk 1]
```diff
@@ -257,6 +257,39 @@ export const explainers = (params, colors) => ({
     text: VERIFIED_EXPLAINER,
     title: lang.t('explain.verified.title'),
   },
+  unverified: {
+    extraHeight: android ? 150 : 120,
```
Comment (estebanmino) [inline src/screens/ExplainSheet.js:261]: following the pictures above, feels like the height in android is too long, action buttons margin with bottom is higher than ios and other explain sheets

**Comment by skylarbarrera** [comment]
android height + button padding has been resolved. there are a few issues that our out of the scope of this ticket:
- Line height issues are an underlying issue that can be seen on other explainers due to this component not fully using the design system (not a quick migration due to all the custom styles + logic we have)
- android sheet heights, especially on Samsung phones is going to be a little stretched due to the outdated SafeAreaInsets package

File: src/screens/ExplainSheet.js [hunk 2]
```diff
@@ -257,6 +263,39 @@ export const explainers = (params, colors) => ({
...
+    secondaryButtonText: lang.t('button.go_back_lowercase'),
+    title: lang.t('explain.unverified.title', {
+      symbol: params?.asset?.symbol,
+    }),
+    stillCurious: (
+      <Text {...getBodyTextPropsWithColor(colors)}>
+        {lang.t('explain.unverified.fragment1')}
+        <Text
+          color={colors?.appleBlue}
+          lineHeight="loose"
```
Comment (estebanmino) [inline src/screens/ExplainSheet.js:282]: removing this line fixes most of the lineHeight problem on android
image

but yeah i see the text lineHeight is not consistent

**Review by estebanmino — APPROVED** [review]
🚀 🚀 🚀 🚀

**Comment by DanielSinclair** [comment]
Spotted a regression for the network explainers where the 'Read More' button disappeared
simulator_screenshot_29EEA3BF-9BFA-4BCE-84C1-DCA4F656EA21

**Comment by skylarbarrera** [comment]
fixed read more button issue: https://cloud.skylarbarrera.com/Screen-Shot-2022-07-01-10-45-11.51.png

**Pull request merged**